### PR TITLE
Fix compilation of round 3 Kyber 90S variants and McEliece

### PR DESCRIPTION
**Pull request by sburton84**

### CODE REVIEW

File: sources_nist_round3/kem/kyber/Reference_Implementation/crypto_kem/kyber1024-90s/symmetric-shake.c [hunk 1]
```diff
@@ -1,60 +0,0 @@
-#include <stddef.h>
```
Comment (jg-cqc) [inline sources_nist_round3/kem/kyber/Reference_Implementation/crypto_kem/kyber1024-90s/symmetric-shake.c:1]: Sean, I'd prefer not to delete anything. These folders are an exact copy of the "Reference Implementation" folder from NIST.  The only files modified to date are build files. Is this file in the way? Do we need to exclude it from the build?

**Review by jg-cqc — CHANGES_REQUESTED** [review]

File: sources_nist_round3/kem/kyber/Reference_Implementation/crypto_kem/kyber1024-90s/symmetric-shake.c (hunk 1 above)
Comment (sburton84) [reply sources_nist_round3/kem/kyber/Reference_Implementation/crypto_kem/kyber1024-90s/symmetric-shake.c:1]: I can add it back if we want, but the Makefile doesn't use it at all, so it serves no purpose.

File: build_algorithms_nist_round3.sh [hunk 2]
```diff
@@ -55,7 +55,6 @@ cd ../../..
 ###################################
 echo "Building McEliece..."
 cd mceliece/Reference_Implementation/kem
-echo "Building McEliece... <======================== TODO: Both McEliece builds are failing. Need fixing. Skipping."
```
Comment (jg-cqc) [inline build_algorithms_nist_round3.sh:58]: Does this mean that Round2 McElice are building ok now?

File: sources_nist_round3/kem/kyber/Reference_Implementation/crypto_kem/kyber512-90s/symmetric-shake.c [hunk 3]
```diff
@@ -1,60 +0,0 @@
-#include <stddef.h>
```
Comment (jg-cqc) [inline sources_nist_round3/kem/kyber/Reference_Implementation/crypto_kem/kyber512-90s/symmetric-shake.c:1]: Similar to previous comment. Can we find a way to leave the sources 100% unaltered?

File: sources_nist_round3/kem/mceliece/Reference_Implementation/kem/mceliece348864/crypto_hash.h [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /* This file uses SHAKE256 implemented in the Keccak Code Package */
 
-#include <libkeccak.a.headers/SimpleFIPS202.h> 
+#include <../../libkeccak.a.headers/SimpleFIPS202.h>
```
Comment (jg-cqc) [inline sources_nist_round3/kem/mceliece/Reference_Implementation/kem/mceliece348864/crypto_hash.h:3]: Can we add this path to the makefile/build system instead?

**Review by jg-cqc — CHANGES_REQUESTED** [review]

File: sources_nist_round3/kem/kyber/Reference_Implementation/crypto_kem/kyber1024-90s/symmetric-shake.c (hunk 1 above)
Comment (jg-cqc) [reply sources_nist_round3/kem/kyber/Reference_Implementation/crypto_kem/kyber1024-90s/symmetric-shake.c:1]: I would like to leave all of the sources untouched if possible - as if we were downloading/pulling the code from the net at build time (like we do with libOQS).

**Comment by sburton84** [comment]
Closing, as these changes don't quite work and the effort would be better spent converting to using libOQS which has a more correct/secure implementation.

**Pull request closed without merging**